source/Account.cpp

```diff
@@ -188,6 +188,7 @@ string Account::Step(int64_t assets, int64_t salaries, int64_t maintenance)
 	// or skipped completely (accruing interest and reducing your credit score).
 	int64_t mortgagesPaid = 0;
 	int64_t finesPaid = 0;
+	int64_t sharesPaid = 0;
```

Comment:
```suggestion
	int64_t profitSharesPaid = 0;
```

This may seem nitpicky, but I've found that naming things exactly what they are makes the code way easier to understand down the track.